### ISSUE TITLE
Rubocop producing warning about new unconfigured cops

### ISSUE THREAD

**Issue by Joseph-Burke** [microverseinc/linters-config#125]
Using the `.rubocop.yml` and `.github/workflows/linters.yml` files from this linters-config/ror repository produces this warning for me in the terminal when running rubocop. It also means that there are a number of offences that are not picked up until the GitHub Actions CI, which is apparently running cops that are not present when I'm linting locally. Am I doing something wrong or do these config files need updating?

```
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that can also opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

Layout/SpaceBeforeBrackets: # (new in 1.7)
  Enabled: true
Lint/AmbiguousAssignment: # (new in 1.7)
  Enabled: true
Lint/DeprecatedConstants: # (new in 1.8)
  Enabled: true
Lint/DuplicateBranch: # (new in 1.3)
  Enabled: true
Lint/DuplicateRegexpCharacterClassElement: # (new in 1.1)
  Enabled: true
Lint/EmptyBlock: # (new in 1.1)
  Enabled: true
Lint/EmptyClass: # (new in 1.3)
  Enabled: true
Lint/LambdaWithoutLiteralBlock: # (new in 1.8)
  Enabled: true
Lint/NoReturnInBeginEndBlocks: # (new in 1.2)
  Enabled: true
Lint/NumberedParameterAssignment: # (new in 1.9)
  Enabled: true
Lint/OrAssignmentToConstant: # (new in 1.9)
  Enabled: true
Lint/RedundantDirGlobSort: # (new in 1.8)
  Enabled: true
Lint/SymbolConversion: # (new in 1.9)
  Enabled: true
Lint/ToEnumArguments: # (new in 1.1)
  Enabled: true
Lint/TripleQuotes: # (new in 1.9)
  Enabled: true
Lint/UnexpectedBlockArity: # (new in 1.5)
  Enabled: true
Lint/UnmodifiedReduceAccumulator: # (new in 1.1)
  Enabled: true
Style/ArgumentsForwarding: # (new in 1.1)
  Enabled: true
Style/CollectionCompact: # (new in 1.2)
  Enabled: true
Style/DocumentDynamicEvalDefinition: # (new in 1.1)
  Enabled: true
Style/EndlessMethod: # (new in 1.8)
  Enabled: true
Style/HashExcept: # (new in 1.7)
  Enabled: true
Style/IfWithBooleanLiteralBranches: # (new in 1.9)
  Enabled: true
Style/NegatedIfElseCondition: # (new in 1.2)
  Enabled: true
Style/NilLambda: # (new in 1.3)
  Enabled: true
Style/RedundantArgument: # (new in 1.4)
  Enabled: true
Style/SwapValues: # (new in 1.1)
  Enabled: true
For more information: https://docs.rubocop.org/rubocop/versioning.html
```

**Comment by Joseph-Burke** [comment]
I was in fact doing something wrong which meant that offences were not being autocorrected - I was using a --fix flag instead of rubocop's -A.

However, the warnings are still there, so I'll leave this issue open.

**Comment by euqueme** [comment]
Hello @Joseph-Burke :wave: thank you for opening this issue :raised_hands:  please ignore these warnings as the mandatory required rules are the ones run by GHA and not locally. 

Does it make sense? are the GHA picking up offenses your local rubocop isn't? if so please describe them, otherwise let's close this issue thank you :pray:

**Comment by bolah2009** [comment]
We can close this now with https://github.com/microverseinc/linters-config/pull/126, thanks!
